app/views/programs/_program.html.erb

```diff
@@ -1,104 +1,70 @@
-<h1><%= program.title %></h1>
-
-<div class="grid grid-cols-1 md:grid-cols-3 place-content-between">
-
-  <div class="mx-2">
-    <div class="text_small flex-grow">
-      <%= program.subject.upcase %> <%= program.catalog_number.titleize %> - <%= program.class_section.titleize %>
+<div class="flex flex-col lg:flex-row mx-2 gap-4">
+  <div class="flex-none">
+    <div class="text_small">
+      <%= program.display_name %>
     </div>
 
     <div class="py-2">
-      <label class="text_field_label flex sm:mt-0 sm:col-span-2">
-      Term
+      <label class="text_field_label">
+      Term hell
```

Comment:
is this meant to be a comment?

Reply:
No, I wanted to be sure that the correct partial was displayed and forgot to remove it. Fixed